experimental/GModule/GModule.jl

```diff
@@ -878,6 +911,27 @@ function hom_base(C::T, D::T) where T <: GModule{<:Any, <:Generic.FreeModule{<:F
   return b
 end
 
+function hom_base(C::T, D::T) where T <: GModule{<:Any, <:Generic.FreeModule{<:AlgClosureElem{<:FinField}}}
```

Comment:
we'll have to change the signature to replace FreeModule -> FPModule
This is done in the CF-WorkInProgress branch. Feel free to ignore this at this point